### ISSUE TITLE
response from a move

### ISSUE THREAD

**Issue by danthompson** [tinney/capture-the-flag#10]
{
  x: 3
  y: 3
  has_peg: true|false
  has_flag: true|false
  team_flag: [100,200]
  opponent_flag: []
  players [[4,4][5,4]]
}

**Comment by tinney** [comment]
Ok I think this is done:  Here are the 3 shapes: 

`{"player"=>{"has_peg"=>true, "x"=>5, "y"=>9, "has_flag"=>false, "is_in_base"=>false}, "flag"=>{"revealed"=>false, "held"=>false, "x"=>nil, "y"=>nil}, "opponents"=>[]}`

`{"player"=>{"has_peg"=>true, "x"=>5, "y"=>9, "has_flag"=>false, "is_in_base"=>false}, "flag"=>{"revealed"=>true, "held"=>false, "x"=>1, "y"=>2}, "opponents"=>[]}`

`{"player"=>{"has_peg"=>true, "x"=>5, "y"=>9, "has_flag"=>true, "is_in_base"=>false}, "flag"=>{"revealed"=>true, "held"=>true, "x"=>5, "y"=>9}, "opponents"=>[{"has_peg"=>true, "x"=>6, "y"=>9, "has_flag"=>false, "is_in_base"=>true}]}`

**Comment by danthompson** [comment]
Nice. In your examples I have a couple questions;
 - who's flag does the `flag` object expose? If it's the opponents, does the player not know where their flag is located then?
 -  the `is_in_base` property is representing the row where the flag is, correct? If so, should we similarly also expose a property indicating the being in the team's zone? e.g., `is_in_safe_zone`, `is_in_team_zone`, `is_in_zone` or something?

Thoughts?

**Comment by tinney** [comment]
Opponents.  The player doesn't know where their flag is and can't ever enter the "zone" that their flag is located in.  

`is_in_base` means you're on your side vs your opponent's side.  Happy to update to `is_in_safe_zone` or one of the others you suggested above if it makes it more clear.

**Comment by danthompson** [comment]
Yeah, I'd vote for something else other than `is_in_base`. `is_in_safe_zone` works I think.
